ts/src/cryptocom.ts

```diff
@@ -1192,31 +1195,49 @@ export default class cryptocom extends Exchange {
             clientOrderId = broker + this.uuid22 ();
         }
         request['client_oid'] = clientOrderId;
+        let marketType = undefined;
+        let marginMode = undefined;
+        [ marketType, params ] = this.handleMarketTypeAndParams ('createOrder', market, params);
+        [ marginMode, params ] = this.customHandleMarginModeAndParams ('createOrder', params);
+        if ((marketType === 'margin') || (marginMode !== undefined)) {
+            request['spot_margin'] = 'MARGIN';
+        } else if (marketType === 'spot') {
+            request['spot_margin'] = 'SPOT';
+        }
+        const stopPrice = this.safeNumberN (params, [ 'stopPrice', 'triggerPrice', 'ref_price' ]);
+        if (stopPrice !== undefined) {
+            request['ref_price'] = this.priceToPrecision (symbol, stopPrice);
+        }
+        const timeInForce = this.safeStringUpper2 (params, 'timeInForce', 'time_in_force');
+        if (timeInForce !== undefined) {
```

Comment:
We could do that but I'm not sure if we should here because the exchange specific param `time_in_force` wouldn't be supported and post only needs to request `exec_inst` and be GOOD_TILL_CANCEL for this endpoint
https://exchange-docs.crypto.com/exchange/v1/rest-ws/index.html#private-create-order